app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CommentsController < ApplicationController
+  def create
+    @comment = Comment.new(
+      text: params[:text],
+      post: Post.find(params[:post_id]),
+      author: User.find(params[:user_id])
+    )
```

Comment:
[OPTIONAL] I think your app will be safer if it uses strong parameters. You can check this [link](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides) out